### DISCUSSION TITLE
While playing the daily challenge you cannot see any leaderboards

### DISCUSSION THREAD

**Discussion by Tenexxt** [ppy/osu#29929] [Q&A]
Basically, you can't see any leaderboard when its enabled while playing since the game doesn't know what to pick.
I think the way to do this would be just showing the top50 like when you have global leaderboards shown, and i don't think it would break if someone would change the top50, since the game loads the leaderboard changes after you end your play, so it would just be outdated.
If this can't be an option since some would perfer to see other leaderboards instead, i already made a post about in-game leaderboard switch (#29386)

**Comment by bdach** [comment]
> you can't see any leaderboard when its enabled while playing since the game doesn't know what to pick

That's... not... how anything works.

It's just not implemented.

**Reply by Tenexxt** [reply]
Yea that makes sense, I just thought that since there are multiple leaderboards to pick you just settled on none, or just didn't think about it when making the dailies.

Either way if you ever planned on implementing it, then you have this as an idea I suppose
